File: py/src/formulas/Root.py
class Root:
    def __init__(self):
        self.data = [None] * 11
        for i in range(2, 11):
            self.data[i] = self.get(i, 10)

    def get(self, val: float, digit_: int = 10) -> float:
        digit = digit_ or 10
        return float(f"{val ** 0.5:.{digit}f}")

File: js/src/formulas/Root.ts
export default class Root {
  private data = [] as number[]

  constructor () {
    for (let i = 2; i <= 10; i++) {
      this.data[i] = this.get(i, 10)
    }
  }

  get (val: number, digit_: number) {
    const digit = digit_ || 10
    return parseFloat((val ** 0.5).toFixed(digit))
  }
}
